documentation/bolt_project_directories.md

```diff
@@ -90,8 +100,15 @@ The default paths for all Bolt configuration, code, and data are relative to the
 |[`bolt.yaml`](bolt_configuration_reference.md)|Contains configuration options for Bolt.|
 |`hiera.yaml`|Contains the Hiera config to use for target-specific data when using `apply`.|
 |[`inventory.yaml`](inventory_file_v2.md)|Contains a list of known targets and target specific data.|
-|[`project.yaml`](bolt_configuration_reference.md#project_configuration_options)|Contains configuration for the Bolt project.  The project.yaml file contains a whitelist of tasks and plans that you can use to limit the output from the `bolt [plan|task] show` command.|
+|[`project.yaml`](bolt_configuration_reference.md#project_configuration_options)|Contains configuration for the Bolt project.  The `project.yaml` file contains a whitelist of tasks and plans that you can use to limit the output from the `bolt [plan|task] show` command.|
 |[`Puppetfile`](bolt_installing_modules.md#)|Specifies which modules to install for the project.|
 |[`modules/`](bolt_installing_modules.md#)|The directory where modules from the `Puppetfile` are installed. In most cases, do not edit these modules locally.|
 |[`site-modules`](bolt_installing_modules.md)|Local modules that are edited and versioned with the project directory.|
 |`data/`|The standard path to store static Hiera data files.|
+
+## Simplified directory structure
+
+If you're working on a new project and you want to use a simplified directory
+structure, you can load your plan and task content directly from your project as
+if it was a module. For more information, see
+[Directory structure](./directory_structure.md#simplified-directory-structure)
```

Comment:
I'm a little concerned that this could convey the wrong impression on what projects are meant for. A simplified directory structure is definitely a benefit of projects, but the intent is that project-level content should be application-specific, while any content being written as part of a reusable library should still go in a module.

Reply:
Yeah agreed. The use-case wasn't clear to me until I watched your demo TBH. I'm starting to think this doc needs its own page